apps/web/components/apps/AppCard.tsx

```diff
@@ -1,40 +1,89 @@
 import Link from "next/link";
+import { useEffect, useState } from "react";
 
 import { trpc } from "@calcom/trpc/react";
+import { App as AppType } from "@calcom/types/App";
+import { Button, SkeletonButton } from "@calcom/ui";
 import Badge from "@calcom/ui/Badge";
-import Button from "@calcom/ui/Button";
 
 interface AppCardProps {
   logo: string;
   name: string;
+  type: AppType["type"];
   slug?: string;
   category?: string;
   description: string;
   rating: number;
   reviews?: number;
   isProOnly?: boolean;
+  categories: string[];
+  isGlobal?: AppType["isGlobal"];
 }
 
 export default function AppCard(props: AppCardProps) {
   const { data: user } = trpc.useQuery(["viewer.me"]);
+  const [installedAppCount, setInstalledAppCount] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    async function getInstalledApp(appCredentialType: string) {
+      const queryParam = new URLSearchParams();
+      queryParam.set("app-credential-type", appCredentialType);
+      try {
+        const result = await fetch(`/api/app-store/installed?${queryParam.toString()}`, {
```

Comment:
We're looking into migrating api endpoints into `tRPC` routers. What do you think @alannnc since you own that endpoint?

Reply:
@Udit-takkar

Reply:
I think you mean @leog , since I don't remember that specific endpoint.

Reply:
@leog @zomars  any update on this ?

Reply:
I'm actually working on migrating App Store to v2 through this issue #3777 so I'm not sure if we should continue with the effort to change the UI according to this issue, @Jaibles?

Reply:
Ok no problem just close this PR if not needed.